onvif/proxy/lib/WSDiscovery10/Typemaps/WSDiscovery.pm

```diff
@@ -24,6 +24,11 @@ our $typemap_1 = {
                'ProbeMatches/ProbeMatch/EndpointReference' => 'WSDiscovery10::Types::EndpointReferenceType',
                'ProbeMatches/ProbeMatch/EndpointReference/ReferenceProperties' => 'WSDiscovery10::Types::ReferencePropertiesType',
                'ProbeMatches/ProbeMatch/EndpointReference/PortType' => 'WSDiscovery10::Types::AttributedQName'
```

Comment:
When I run this I get complaints about the comma missing at the end of this line